test/Typesense.Tests/TypesenseClientTests.cs

```diff
@@ -1,10 +1,10 @@
-using FluentAssertions;
-using FluentAssertions.Execution;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text.Json.Serialization;
 using System.Threading.Tasks;
+using FluentAssertions;
+using FluentAssertions.Execution;
 using Xunit;
```

Comment:
Can you undo the changes here so they're alphabetically sorted like before the changes?

Reply:
Ohh now I see what happened, the diff was weird before :D

I removed that from the commit.